crates/tests/src/e2e/ledger_tests.rs

```diff
@@ -2556,3 +2557,32 @@ fn masp_txs_and_queries() -> Result<()> {
 
     Ok(())
 }
+
+/// Test localnet genesis files with `namada node utils test-genesis` command.
+#[test]
+fn test_localnet_genesis() -> Result<()> {
+    let loc = format!("{}:{}", std::file!(), std::line!());
```

Comment:
can't this be implemented as an integration test?

Reply:
we don't have node cmds in integration tests :/